sdk/src/trace/tracer.h

```diff
@@ -1,8 +1,8 @@
 #pragma once
 
+#include "opentelemetry/sdk/trace/recorder.h"
```

Comment:
I'm not 100% sure about the purpose of `sdk/include`, but I guess class that are supposed to be implemented by vendors should be placed there too?